src/views/Nft/market/Collection/Traits/index.tsx

```diff
@@ -23,7 +25,11 @@ const Traits = () => {
     <>
       <Header collection={collection} />
       <Container py="40px">
-        <PancakeBunniesTraits collectionAddress={address} />
+        {collectionAddress === pancakeBunniesAddress ? (
+          <PancakeBunniesTraits collectionAddress={address} />
+        ) : (
+          <CollectionTraits collectionAddress={collectionAddress} />
```

Comment:
nitpicky AF but having `address` and `collectionAddress` here feels weird, they are the same in every case as I understand 😄  Except `address` can be empty if user enters URL with non-existent collection address... anyway, just a thought

Reply:
Nah, you're right.